### ISSUE TITLE
children limited to 20 results

### ISSUE THREAD

**Issue by lgatto** [lgatto/rols#25]
It has been reported that children only return up to 20 results. Need to check whether there isn't a hard coded limit to the number of results.
```r
> library("rols")
> trms <- terms("efo")
> cn <- children(trms[["efo:0005140"]])
> cn
Object of class 'Terms' with 20 entries
  From the EFO ontology
   EFO:0006812, EFO:0004208 ... EFO:0007292, EFO:1001511 
```
although this term should have more children.

**Comment by lgatto** [comment]
Indeed, the pagesize limit is 20. A fix will be pushed within 24 hours.

**Comment by lgatto** [comment]
Closed in 2611f4637f1e4a3b51c4b494021fd44e26d4f6e2
